src/components/grid-aware/Footer/Footer.module.css

```diff
@@ -0,0 +1,257 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-rows: 120px 380px 78px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    text-align: center;
+  }
+
+  .gridParent {
+    column-gap: var(--grid-column-gap);
+    display: grid;
+    grid-template-rows: 190px auto auto;
+  }
+}
+
+/* Top Row */
+
+.gridTopRow {
+  align-content: center;
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  grid-row: 1;
```

Comment:
If you end up splitting each of the rows into their own separate component and file, like I suggested in an earlier comment, then I think it may neither be necessary nor useful to use CSS grid for the top row. Nothing in it is aligned to the grid at all.

Reply:
ok